docs/4-roles-expectations.md

```diff
@@ -14,33 +14,49 @@ It should be noted that the Guild as an entity does not directly manage or parti
 
-- Fully open source, ie. available for forking, modification and redistribution
-- Continuous for at least 6 months ahead of inclusion
-- Released under permissive licenses
-- Part of the protocol, not built on top of it, e.g. a specific client implementation vs. an application
-- Beneficial to the broader Ethereum community 
-- Dedicated to a permissionless, open access Ethereum
+Qualifying contributions _must_ be:
 
-It is possible for eligible individuals to work on a team which has a token, has VC backing or gives equity. However, members should expect these guidelines to change over time, i.e. become more restrictive in some places and more permissive in others.
+- Fully open source, ie. both “source available” and free to fork, modify, redistribute
+- The full focus of the individual (anything less receives a partial weighting, see [6.3 weighting](https://protocol-guild.readthedocs.io/en/latest/6-guidelines-for-regular-operation.html#weighting))
+- Continuous for at least 6 months ahead of inclusion and ongoing
 
-Though Ethereum is a highly technical field, it is clear that the work of non-developers around the core protocol is also crucial and should be eligible. This may include people affiliated with the All Core Devs call, those coordinating ecosystem upgrades, and anyone helping to coevolve the protocol alongside the community.
+Qualifying contributions _must_ target at least one of the following projects/areas:
 
-There may also be contributors to Protocol Guild itself whose work is significant enough to consider for membership. This work may include: 
+1. Ethereum L1 core protocol maintenance and development
 
-- general comms/public presentations to improve awareness or explain the mechanism
-- fundraising from the broader ecosystem (eg. funding programs, individuals, larger entities who depend on Ethereum)
-- research related to the evolution of the Guild itself: smart contract architecture, surveys to members, documentation
-- internal maintenance for the Guild membership: helping with curation processes, onboarding/offboarding members
+- Production client implementations: [Erigon](https://github.com/ledgerwatch/erigon), [Geth](https://github.com/ethereum/go-ethereum), [Hyperledger Besu](https://github.com/hyperledger/besu), [Lighthouse](https://github.com/sigp/lighthouse), [Lodestar](https://github.com/ChainSafe/lodestar), [Nethermind](https://github.com/NethermindEth/nethermind), [Nimbus](https://github.com/status-im/nimbus-eth2), [Prysm](https://github.com/prysmaticlabs/prysm), [Teku](https://github.com/ConsenSys/teku)
+- Client Testing/security/infra which supports these implementations: [ethereum/tests](https://github.com/ethereum/tests) [ethereum/execution-spec-tests](https://github.com/ethereum/execution-spec-tests)
+- Coordination related to upgrades and maintenance: [ethereum/pm](https://github.com/ethereum/pm)
+
+2. Research and implementation experiments related to potential protocol changes/refinements
```

Comment:
Initially I wasn't clear that this was the category under which research falls because I read it as "(research and implementation) experiments ~ research experiments and implementation experiments", and the examples are mostly for the implementation or teams that are in-between. Perhaps it would be more legible to add an example of a research team too?

Reply:
the old/easy way would be to list RIG 😛 what else could we add?

- fork choice optimizations
- single slot finality
- SLLE
- resource pricing
- mechanism design
- monetary incentives
- PBS

Reply:
Since the new framework appears to be more project-based, yes the list you provided makes sense. I would remove monetary incentives (that sounds like mechanism design, unless you mean monetary policy) and I would add cryptography, maybe group fork choice/ssf under "consensus"? So in a way the three EF research teams (consensus, cryptography, mechanism design)

Reply:
added these suggestions, thanks!